katapult_test.go

```diff
@@ -17,6 +18,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var testDefaultBaseURL = &url.URL{Scheme: "https", Host: "api.katapult.io"}
+
 func Test_apiClient_NewRequestWithContext(t *testing.T) {
```

Comment:
Test function names should be updated to reflect the struct being renamed from `apiClient` to `Client`:

```suggestion
func TestClient_NewRequestWithContext(t *testing.T) {
```

And also the `Test_apiClient_Do()` function should be renamed to `TestClient_Do()`, but there's no diff changes around that function definition here, so I can't add a comment and code suggestion to it, so you'll have to do that one manually :)